ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -80,11 +80,17 @@ cabalFileLibraries ghcFlavor =
     ]
 
 -- C-preprocessor "include dirs" for 'ghc-lib-parser'.
-ghcLibParserIncludeDirs :: GhcFlavor -> [FilePath]
-ghcLibParserIncludeDirs ghcFlavor =
+ghcLibParserIncludeDirs :: Bool -> GhcFlavor -> [FilePath]
+ghcLibParserIncludeDirs _forDepends ghcFlavor =
```

Comment:
It looks like this _forDepends parameter is no longer used

Reply:
i introduced this generalizationresearching solutions to this PR. in the end we don't use it but it makes sense to leave it plumbed through now that it's there. chances are it will prove useful in the future.